### PR TITLE
✨♻️ SsmPage containers

### PR DESCRIPTION
**Pull request by alex-wilmer**
Containerizes remaining SSM page components, removes SSM.js and pulls thing into SsmPage.js, fixes some queries, removes unused vars / imports

### CODE REVIEW

**Comment by alex-wilmer** [comment]
getting an issue with node query getting overwritten somewhere

**Comment by cy** [comment]
https://github.com/NCI-GDC/portal-ui/pull/1527 edited the same file going to have conflicts

**Comment by alex-wilmer** [comment]
requires https://github.com/NCI-GDC/gdcapi/pull/758

File: modules/node_modules/@ncigdc/containers/SsmExternalReferences.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  />
+));
+
+export const SsmExternalReferencesQuery = {
+  fragments: {
+    node: () => Relay.QL`
+      fragment on Ssm {
+        cosmic_id
+        consequence {
+          hits(first: 99) {
```
Comment (cy) [inline modules/node_modules/@ncigdc/containers/SsmExternalReferences.js]: could put a `{"op": "NOT", "content": {"field": "consequence.transcript.annotation.dbsnp_rs", "value": "missing"}}` filter here and then `first: 1`? The UI only shows one

**Pull request merged**